phylogenetic/Snakefile

```diff
@@ -8,11 +8,14 @@ workdir: workflow.current_basedir
 # Use default configuration values. Override with Snakemake's --configfile/--config options.
 configfile: "defaults/config.yaml"
 
-SEGMENTS = ["l", "s"]
+segments = ["L", "S"]
```

Comment:
Flagging that the `l` -> `L` and `s` -> `S` change will require updating the [nextstrain.org manifest for lassa](https://github.com/nextstrain/nextstrain.org/blob/77ebbcd991a3d59f3cc39da45b4b2940c4390c43/data/manifest_core.json#L413-L414)

Reply:
Geh, good to know. I can revert `L` -> `l` in phylogenetic and back up to ingest as that may be a more consistent-with-history solution.